test/support/capybara.rb

```diff
@@ -21,6 +21,10 @@ def switch_domain(domain)
   xpath {|text| "//tr[./td[text() = '#{text}']]" }
 end
 
+Capybara.add_selector(:row_link) do
```

Comment:
Since changing the description of xpath of ": row" corresponded, ": row_link" has been deleted.